src/FastRandom.cpp

```diff
@@ -1,121 +1,66 @@
 
 // FastRandom.cpp
 
-// Implements the cFastRandom class representing a fast random number generator
 
 #include "Globals.h"
 #include "FastRandom.h"
 
+#include <mutex>
 #include <random>
 
+#ifdef __clang__
+	#pragma clang diagnostic ignored "-Wglobal-constructors"
+#endif
+
```

Comment:
Usually we try to fix whatever the warnings are hinting at, unless the fix is too unwieldy. Better than disabling the warning on a whole file where it could actually hide something important.

In this case, wouldn't it work if you refactored the two globals into `static` variables inside the only function that uses them?